bin/rbenv-update

```diff
@@ -1,23 +1,24 @@
 #!/usr/bin/env bash
-# Usage: rbenv update [--noop]
+# Usage: rbenv update [--noop|--quiet|--silent]
 # Summary: Updates to latest version of rbenv and plugins from git.
 set -e
 [ -n "$RBENV_DEBUG" ] && set -x
 
+# Parse command line flags
 noop=""
```

Comment:
Am not familiar with that, can you elaborate? (Been using fish as my main shell for a couple of years now)